packages/server/Dockerfile

```diff
@@ -27,7 +27,7 @@ COPY scripts/integrations/oracle/ scripts/integrations/oracle/
 RUN /bin/bash -e scripts/integrations/oracle/instantclient/linux/x86-64/install.sh 
 
 COPY package.json .
-RUN yarn install --frozen-lockfile --production=true
+RUN yarn install --production=true
```

Comment:
Just a general question on the `install` - In local dev how does this work with dependencies? 
I take it now that we're building a bundled dist file the bb deps will be included there. Does the install just pull down a fixed version from npm that's ignored?